file/provider/util.go

```diff
@@ -249,6 +249,51 @@ func FromProviderToLibVolume(vpcVolume *models.Share, logger *zap.Logger) (libVo
 		libVolume.Az = vpcVolume.Zone.Name
 	}
 	libVolume.CRN = vpcVolume.CRN
+
+	var respAccessPointlist = []provider.VolumeAccessPoint{}
+
+	shareTargetlist := vpcVolume.ShareTargets
+	if shareTargetlist != nil && len(*shareTargetlist) > 0 {
+		for _, shareTargetItem := range *shareTargetlist {
+			volumeAccessPointResponse := FromProviderToLibVolumeAccessPoint(&shareTargetItem, logger)
+			respAccessPointlist = append(respAccessPointlist, *volumeAccessPointResponse)
+		}
+	}
+
+	libVolume.VolumeAccessPoints = &respAccessPointlist
+	return
+}
+
+// FromProviderToLibVolumeAccessPoint converting vpc provider share target type to generic lib volume accessPoint Type
+func FromProviderToLibVolumeAccessPoint(vpcShareTarget *models.ShareTarget, logger *zap.Logger) (libVolumeAccessPoint *provider.VolumeAccessPoint) {
+	logger.Debug("Entry of FromProviderToLibVolumeAccessPoint method...")
```

Comment:
debug will never be shown in the log can be add as info

Reply:
This pattern is followed in block so carried over. I though we don't need to show this entry exit until we want to enable debug.